examples/api/lib/widgets/sliver/sliver_decoration.0.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const SliverDecorationExampleApp());
+
+class SliverDecorationExampleApp extends StatelessWidget {
+  const SliverDecorationExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(title: const Text('SliverDecoration Sample')),
+        body: const SliverDecorationExample(),
+      ),
+    );
+  }
+}
+
+class SliverDecorationExample extends StatelessWidget {
+  const SliverDecorationExample({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return CustomScrollView(
+      slivers: <Widget>[
+        SliverDecoration(
```

Comment:
Can we have two to demonstrate multiple decorated slivers in one scroll view? 🤩

Reply:
Added another decorated sliver.